src/groupByKey.ts

```diff
@@ -1,7 +1,19 @@
+interface Obj {}
+
 interface Item {
-  [key: string]: any;
+  [key: string]: Obj[];
```

Comment:
```suggestion
  [key: string]: any;
```
Пропиши інтерфейс для групи айтемів і використовуй його

Reply:
Виправив, але вже другий тиждень не вдається відправити на ре-ревью,просто не натискається кнопка.Мабуть якийсь баг в Git.